controller/api/management/commands/load_db_state_to_etcd.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand
+from api.models import Key, App, Domain, Certificate, Config
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        print "Publishing DB state to etcd..."
+        for M in (Key, App, Domain, Certificate, Config):
```

Comment:
I would use `model` and `obj` as local var names instead of `M` and `o`. Brief but descriptive (and lower-case) is pythonic, it's only in Go that we get to use single-char names, except for special cases `i` and `x`.